components/org.wso2.carbon.identity.remotefetch.core.ui/src/main/resources/web/remotefetch-core/delete-remotefetch-config.jsp

```diff
@@ -22,22 +22,14 @@
 <%
     
     if (request.getParameter("id") != null) {
-        int id = -1;
-        
+        String id = request.getParameter("id");
```

Comment:
Better to move this "id" to a constant

Reply:
This one time used a local parameter, Can you please explin we need to move as constant in this JSP file?

Reply:
Resolved